testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.openapi

```diff
@@ -0,0 +1,61 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "components": {
+    "schemas": {
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789___1_-_-_1": {
+        "additionalProperties": false,
+        "properties": {},
+        "type": "object",
+        "description": "Short description. TODO: Fix bug with long description, task SERV-374"
```

Comment:
The same comment, as before. Why did you change the test? It is ok if it fails until the bug is fixed.

Reply:
fixed, but '#' inside single line annotation will stop tests after uncomment todo